functions/package.json

```diff
@@ -6,7 +6,7 @@
     "deploy": "firebase deploy --only functions",
     "lint": "eslint .",
     "logs": "firebase functions:log",
-    "serve": "firebase serve --only functions",
+    "serve": "firebase serve --only functions --port 5054",
```

Comment:
This gets things working on macOS again. Port 5000, the Firebase default, is now reserved for Airdrop.